Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Dockerfile
-FROM node:10-alpine
+FROM node:12.16.2-alpine
```

Comment:
can this be read from the nvmrc directly to not have a duplication?

Reply:
You're right it's totally possible, once `renovate bot` will be installed it won't be an issue but it's a good idea for now.